pkg/gofr/datasource/pubsub/nats/client.go

```diff
@@ -33,6 +33,8 @@ type messageHandler func(context.Context, jetstream.Msg) error
 
 // Connect establishes a connection to NATS and sets up jStream.
 func (c *Client) Connect() error {
+	c.logger.Debugf("connecting to NATS server at %v", c.Config.Server)
```

Comment:
This log is a repeated in the Connect method called here:

```go
connManager := NewConnectionManager(c.Config, c.logger, c.natsConnector, c.jetStreamCreator)
	if err := connManager.Connect(); err != nil {
		c.logger.Errorf("failed to connect to NATS server at %v: %v", c.Config.Server, err)
		return err
	}
```


And inside the Connect method we also have this log:



```go
// Connect establishes a connection to NATS and sets up JetStream.
func (cm *ConnectionManager) Connect() error {
	cm.logger.Debugf("Connecting to NATS server at %v", cm.config.Server)

	opts := []nats.Option{nats.Name("GoFr NATS JetStreamClient")}

	if cm.config.CredsFile != "" {
		opts = append(opts, nats.UserCredentials(cm.config.CredsFile))
	}

	connInterface, err := cm.natsConnector.Connect(cm.config.Server, opts...)
	if err != nil {
		cm.logger.Errorf("failed to connect to NATS server at %v: %v", cm.config.Server, err)

		return err
	}
```

We need to remove either of them